Login_Forms/login-animated/style.css

```diff
@@ -0,0 +1,161 @@
+
+#login_box {
+    width: 330px;
+    height: 230px;
+    background-color: #2b1710;
+    border-radius: 10px;
+    margin: auto;
+    margin-top: 100px;
+    padding: 20px;
+    box-shadow: 5px 5px 5px grey, -5px 5px 5px grey;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+#login_box h1 {
+    color: #fff;
+    text-align: center;
+}
+
+#login_box .txt_field label{
+    color: #fff;
+    position: absolute;
+    transform: translateY(-250%);
+}
+
+#login_box .txt_field input{
+    border: none;
+    outline: none;
+    width: 100%;
+    height: 20px;
+    color: #fff;
+    background-color: #2b1710;
+    margin-bottom: 23px;
+    border-bottom: solid 2px #2691d9;
+}
+
+#login_box .txt_field input:focus ~ label,
+#login_box .txt_field input:valid ~ label{
+    transform: translateY(-400%);
+    color: #2691d9;
+    font-size: 13px;
+    transition: 0.4s;
+}
+
+#login_box .submit {
+    transform: translateX(-10%);
+    padding: 0 0;
+    position: relative;
+    width: 30%;
+    height: 30px;
+}
+
+#login_box .submit .border{
+    padding: 2px 2px;
+    height: 100%;
+    width: 100%;
+    position: relative;
+    overflow: hidden;
+}
+
+#login_box .submit #login_box_submit{
```

Comment:
add some margin to the left and top of the submit button
![image](https://user-images.githubusercontent.com/64855541/147643259-02758a76-2b58-48ca-b149-5fec6fc29cbd.png)